WarmTransfer.Web.Tests/app.config

```diff
@@ -4,7 +4,7 @@
     <add key="TwilioAccountSid" value="account-sid" />
     <add key="TwilioAuthToken" value="auth-token" />
     <add key="TwilioPhoneNumber" value="twilio-phone-number" />
-    <add key="Domain" value="example.com"/>
```

Comment:
You could probably get rid of this in the settings by doing something like this when building the url
https://github.com/TwilioDevEd/clicktocall-csharp/blob/next-gen/ClickToCall.Web/Controllers/CallCenterController.cs#L61-L62